src/com/kodilla/ui/Board.java

```diff
@@ -26,31 +28,34 @@
     private static final int GAME_BOARD_MAX_ROWS = 3;
     private static final int GAME_BOARD_HGAP = 15;
     private static final int GAME_BOARD_VGAP = 15;
-    private GridPane gameBoard;
-    private BoardPresenter status;
-    private String computerGameMark;
+    private GridPane viewGameBoard;
+    private BoardPresenter presenter;
 
     @Override
     public void start(Stage primaryStage) {
+        createPresenter(primaryStage);
         primaryStage.setTitle(APP_NAME);
         primaryStage.setScene(createScene());
         primaryStage.show();
     }
 
-    public Scene createScene() {
-        gameBoard = new GridPane();
-        gameBoard.setBackground(createBackground());
-        computerGameMark = String.valueOf(FigureType.CIRCLE);
+    private void createPresenter(Stage primaryStage) {
```

Comment:
jak nie zawraca to nie `create..` a może `bind..` albo `prepare..`?

Reply:
done!